src/Organizer/OrganizerLDProjector.php

```diff
@@ -366,6 +374,39 @@ private function newDocument($id)
     }
 
     /**
+     * @param TitleUpdated $titleUpdated
+     * @param Language|null $language
+     */
+    private function applyTitle(
+        TitleUpdated $titleUpdated,
```

Comment:
Om terug te komen op mijn eerste comment, hier is het bv niet vanzelf duidelijk dat TitleTranslated ook mogelijk is. Ik had het pas zelf door als ik terug ging kijken naar waar deze method werd gebruikt en naar TitleTranslated zelf.